### ISSUE TITLE
[WATT] Modifications of HTML file are not visible in preview

### ISSUE THREAD

**Issue by lslachciak** [Samsung/TAU-Design-Editor#115]
Steps:
[1] Run latest WATT
[2] Visit http://localhost:3000/demos?path=1.0%2Fexamples%2Fwearable%2FUIComponents%2Fcontents%2Fcontrols%2Fcheckbox.html in Chromium browser
[3] Run Live Preview
[4] Quit Live preview
[5] Modify some html content
[6] Go to Live preview
Observe: No changes in UI after modification of html source code

**Comment by korneliakobiela** [comment]
1. You should save the document before start design-editor Live Preview.
2. The problem is quite big and causes only into demo mode. In this mode, the preview is only hidden and detached callback isn't fired.

**Comment by lslachciak** [comment]
@korneliakobiela I'm saving document, but issue occurs.
And in fact there is a warning if you forget to save.

**Comment by lslachciak** [comment]
After discussing issue with @korneliakobiela I updated description of issue

**Comment by korneliakobiela** [comment]
I started working on it today. After a few different approaches, I decided to host the original file (instead of temporary-preview.html) for the demo mode. Now I have to solve one more problem, an iframe with preview seems not to refresh in one Design Editor session.

**Comment by korneliakobiela** [comment]
My changes are presented in this PR: #122

**Comment by korneliakobiela** [comment]
Unfortunately, the temporary file for preview `temporary-preview.html` has to stay even in the demo mode. We have some helper solutions in preview which doesn't have to stay in the original app.

**Comment by korneliakobiela** [comment]
After the last update, the only problem with this issue is that the preview iframe seems not to refresh itself after getting the new address. I don't know what is causing this problem and need more time to investigate. Then I think it should work.

Additionally, I fixed the issue with `config.xml`

**Comment by korneliakobiela** [comment]
Today at least I did some first working version of my solution. I committed the knockabout version of the code to my PR. Tomorrow morning I'll refactor, write some tests and send it to review. 

**Technical details below:** 

1. Preview panel was created but wasn't destroyed in any place.
2. Event listeners for this panel were stacked. 
3. After the next call of preview, there were more than one events and event listeners
4. New preview panel was appended to the DOM but was covered by previously generated.
5. User didn't see any changes.
